includes/functions.php

```diff
@@ -2022,21 +2046,21 @@ function tsml_scan_data_source($data_source_url)
 
 				if (count($meetings_updated) > 0) {
 					// Send Email notifying Admins that this Data Source needs updating
-					$message = "Data Source changes were detected during a scheduled sychronization check with this feed: $data_source_url. Your website meeting list details based on the $data_source_name feed are no longer in sync. <br><br>Please sign-in to your website and refresh the $data_source_name Data Source feed found on the Meetings Import & Settings page.<br><br>";
+					$message = "Data Source changes were detected during a scheduled sychronization check with this feed: $data_source_url. Your website meeting list details based on the $data_source_name feed are no longer in sync. <br><br>Please sign-in to your website and update the $data_source_name Data Source feed found on the Meetings Import & Export page.<br><br>";
```

Comment:
is this text current? users don't need to sign in and update now right - the update has already been applied?

Reply:
nevermind, i was confused ;)

Reply:
??? Don't understand this question. This is the Change Detection code for the email sent to the registered change notification addresses. No updates have been applied to the local database yet!